packages/html/src/HTMLTooltip.ts

```diff
@@ -66,6 +95,10 @@ export class HTMLTooltip extends HTMLWidget {
         this.updateTooltipPosition();
     }
 
+    onShowContent(node) {
+        
+    }
+
```

Comment:
Is you goal to let the user add a DOMElement instead of text html?

Reply:
My goal is to allow either one to be used.